cargo-dist/src/host.rs

```diff
@@ -208,7 +212,21 @@ fn release_hosting(_dist: &DistGraph, manifest: &DistManifest, abyss: &Gazenot)
             None
         }
     });
-    tokio::runtime::Handle::current().block_on(abyss.create_releases(releases))?;
+
+    // Tell The Abyss To Release
+    let new_releases =
+        tokio::runtime::Handle::current().block_on(abyss.create_releases(releases))?;
+
+    // Update artifact download URLs with release results
+    for new_release in new_releases {
+        if let Some(new_url) = new_release.release_download_url {
+            manifest.update_release_axodotdev_artifact_download_url(&new_release.package, new_url);
+        }
+    }
+
+    // Update Github Announcement body with new URLs
+    announcement_github(manifest);
```

Comment:
This is failing the test because the new logic is producing an empty GitHub release announcement.

The test in question is producing an announcement for `cargo-dist-schema`. At this point, we've filtered the list of releases down to zero - we're not releasing `cargo-dist`, and `cargo-dist-schema` has no binaries. As a result, `announcement_github()` is being called on a manifest that has zero releases in it. Whereas before you checked the global `self.inner.ci_style`, which would still be valid, here you're checking the manifest's releases and concluding that `announcing_github` should be `false`.

That... might be right, actually? Up to you whether this is actually more correct than the old behaviour.